tsp-output/@azure-tools/typespec-csharp/src/Generated/Audio.cs

```diff
@@ -130,8 +130,8 @@ public virtual async Task<ClientResult<CreateTranscriptionResponse>> CreateTrans
         {
             Argument.AssertNotNull(audio, nameof(audio));
 
-            (BinaryContent content, string contentType, RequestOptions options) = await audio.ToMultipartContentAsync().ConfigureAwait(false);
-            ClientResult result = await CreateTranscriptionAsync(content, contentType, options).ConfigureAwait(false);
+            using MultipartFormDataBinaryContent content = audio.ToMultipartContent();
```

Comment:
In this version, the content-creation is not async, and the call pattern is a bit simplified.

Reply:
https://github.com/Azure/autorest.csharp/issues/4340